### PR TITLE
Fixed some typos. Unified notation to match other entries on resolution. Added two entries to the bibliography.

### PR DESCRIPTION
**Pull request by uhustadt**

### CODE REVIEW

File: Source/mainmatter/LTLClausalResolution.tex [hunk 1]
```diff
@@ -13,9 +13,8 @@
 \calculusAuthor{Ullrich Hustdat}
```
Comment (ceilican) [inline Source/mainmatter/LTLClausalResolution.tex]: There is a typo here, isn't there?

**Comment by ceilican** [comment]
Thanks, @uhustadt !  I found another typo. Have a look on my comment here: https://github.com/ProofSystem/Encyclopedia/pull/73/files .

**Comment by uhustadt** [comment]
Dealt with the author name issue and worked on the entry for Ordered Fine-Grained Resolution for Monodic First-Order Temporal Logic.

**Comment by ceilican** [comment]
Thanks, @uhustadt !

**Pull request merged**